### DISCUSSION TITLE
Ferns & Large Ferns don't drop Straw

### DISCUSSION THREAD

**Discussion by SkyanUltra** [vectorwing/FarmersDelight#995] [Ideas]
This feels like a minor oversight, which is why I'm not writing a bug report for this as much as I'm suggesting that it be added.

Some biomes are very heavy with Fern generation, so I feel like adding them to the list of blocks that drop straw when cut with a knife would be a simple and convenient change to make for those biomes in particular.

**Comment by vectorwing** [comment]
This is partially intentional, as ferns aren't grasses, and straw cannot really be obtained from them traditionally.

That said, this is a "realism vs intuition" problem: it would be convenient and seamless to players if Ferns could also drop Straw during gameplay, despite the explanation above. I could look into this change sometime.
